addons/sale/static/src/js/action_helper/sale_action_helper.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+import { renderToElement } from "@web/core/utils/render";
+
+export class SaleActionHelper extends Component {
+    static template = "sale.SaleActionHelper";
+    static props = ["noContentHelp"];
+    setup() {
+        // url is fixed;
+        this.videoURL = "https://www.youtube.com/embed/N4zw-2t6spk?autoplay=1";
+    }
+
+    async openVideoPreview() {
+        const self = this;
+        this.$modal = $(renderToElement('sale.VideoPreview', {url: this.videoURL}));
```

Comment:
We don't use `jQuery` in OWL
you should import `renderToElement` from render and user it.
No? :thinking:

Reply:
```javascript
 async openVideoPreview() {
    this.modal = renderToElement('sale.VideoPreview', {url: this.videoURL});
    document.body.append(this.modal);
    this.modal.addEventListener('hidden.bs.modal', () => {
        this.modal.remove();
    });
    const modal = new Modal(this.modal);
    modal.show();
}
``` 
You could do this!
Thank you @visp-odoo for helping him